File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/duckdb/JsonType.kt
package dev.typr.foundationskt.docs.duckdb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class JsonType {
    //start
    val jsonType: DuckDbType<Json> = DuckDbTypes.json

    val data: Json = Json("{\"name\": \"DuckDB\"}")
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/duckdb/JsonType.scala
package dev.typr.foundationssc.docs.duckdb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object JsonType:
  //start
  val jsonType: DuckDbType[Json] = DuckDbTypes.json

  val data: Json = new Json("{\"name\": \"DuckDB\"}")
  //stop
